addon/server.js

```diff
@@ -167,7 +130,7 @@ export default class Server {
     let hasFactories = this._hasModulesOfType(config, 'factories');
     let hasDefaultScenario = config.scenarios && config.scenarios.hasOwnProperty('default');
 
-    this.pretender = this.pretender || createPretender(this);
+    this.interceptor = this.interceptor || createInterceptor(this);
```

Comment:
`this.pretender` was basically a public API, it let folks get access to the underlying pretender server. I'm thinking we should add code here to to create a `this.pretender` alias if `this.intercepter instanceof Pretender`.